src/imitation/scripts/config/train_adversarial.py

```diff
@@ -115,7 +116,7 @@ def airl():
 ANT_SHARED_LOCALS = dict(
     total_timesteps=3e7,
     max_episode_steps=500,  # To match `inverse_rl` settings.
-    algorithm_kwargs=dict(shared=dict(disc_batch_size=2048 * 8)),
+    algorithm_kwargs=dict(shared=dict(expert_batch_size=2048 * 4)),
```

Comment:
why do we specify it as `2048 * 4` rather than `8192`?

Reply:
I don't remember. I'll convert this